dags/examples/xpk_example_dag.py

```diff
@@ -63,7 +63,7 @@
   #    "{gcs_bucket.BASE_OUTPUT_DIR}/{gcs_subfolder}/{group_id}-{current_datetime}/"
   test_group_id = "chained_tests"
   gcs_subfolder = f"{test_owner.Team.MULTIPOD.value}/maxtext"
-  with TaskGroup(group_id=test_group_id) as group:
+  with TaskGroup(group_id=test_group_id, prefix_group_id=False) as group:
     shared_gcs_location = name_format.generate_gcs_folder_location(
```

Comment:
Removing the task_group prefix, should we also make the task ID for `generate_gcs_folder_location` unique to support multiple groups per DAG?

Reply:
Good point. Yes, we need. We could override the task id of `generate_gcs_folder_location` to be specific with shared id. Let me update the example and make comments.

Reply:
Update the task id as an example: https://screenshot.googleplex.com/7NY7i7MDbV3dmR2